src/main/java/pl/tomaszdziurko/jvm_bloggers/TimeConstants.java

```diff
@@ -6,4 +6,7 @@
 
     public static final String EVERY_FRIDAY_AT_12_OCLOCK = "0 0 12 * * FRI";
 
+    public static final String EVERY_SIX_HOURS = "0 0 */4 * * *";
+
+    public static final String EVERY_EIGHT_AM_PM_HOURS = "0 0 9,18 * * *";
```

Comment:
`EVERY_EIGHT_AM_AND_PM` would be more clear, `HOURS` does not add anything here

and... `9` is not `EIGHT`